### PR TITLE
kde-apps/kalzium: Add kf5-based ebuild

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

File: kde-apps/kalzium/kalzium-5.9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+src_configure(){
+	# Fix missing finite()
+	[[ ${CHOST} == *-solaris* ]] && append-cppflags -DHAVE_IEEEFP_H
+
+	local mycmakeargs=(
+		-DCMAKE_DISABLE_FIND_PACKAGE_Eigen3=$(usex editor)
+		-DCMAKE_DISABLE_FIND_PACKAGE_AvogadroLibs=$(usex editor)
+		-DCMAKE_DISABLE_FIND_PACKAGE_OpenBabel2=$(usex editor)
+		-DCMAKE_DISABLE_FIND_PACKAGE_OCaml=$(usex solver)
+		-DCMAKE_DISABLE_FIND_PACKAGE_Libfacile=$(usex solver)
```
Comment (johu) [inline kde-apps/kalzium/kalzium-5.9999.ebuild]: cmake-utils_use_find_package is still available in EAPI 6

Comment (a17r) [inline kde-apps/kalzium/kalzium-5.9999.ebuild]: Yes I know, am I required to use it here? This was just the fastest way to convert from -DWITH_*

Comment (johu) [inline kde-apps/kalzium/kalzium-5.9999.ebuild]: It is more readable, that is the reason why it was not removed with EAPI 6. We should not mix ebuild style in kde project.

**Comment by johu** [comment]
Thanks rebased onto kde overlay master.

**Pull request closed without merging**